### PR TITLE
Initial Header

### PR DESCRIPTION
**Pull request by rjg-fizen**
This has the top level navigation and "NEWS" wording.  Sub level heading will come next.  I also need to build the "more" dropdown menu and functionality.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,16 +1,104 @@
...
-</head>
-<body>
-    
+    <link rel="stylesheet" href="./styles/main.css" />
+  </head>
+  <body>
+    <header>
+      <nav id="header-nav" class="center super-wide">
+        <ul class="horizontal-nav">
+          <li></li>
```
Comment (Elybourland) [inline index.html:13]: No content in list bracket.

File: index.html [hunk 2]
```diff
@@ -1,16 +1,104 @@
...
+            </div>
+          </li>
+          <li class="nav-item ">
+            <div class="border-color-purple">
+              <a href="">Culture</a>
+            </div>
+          </li>
+          <li class="nav-item" id="more">
+            <div class="border-color-dark-green">
+              <a href="" class="float-left">More</a><i class="fas fa-sort-down float-left"></i>
```
Comment (Elybourland) [inline index.html:61]: What is <i class>?

Comment (Elybourland) [reply index.html:61]: li class?

File: index.html (hunk 1 above)
Comment (rjg-fizen) [reply index.html:13]: Ya, I'm just using it to set the BBC background image from CSS.

File: index.html (hunk 2 above)
Comment (rjg-fizen) [reply index.html:61]: Thats the down arrow next to "more".  Im using a font-library called fontawesome.  Look it up!

File: index.html (hunk 1 above)
Comment (rjg-fizen) [reply index.html:13]:
```css
/* BBC Image */
#header-nav > ul > li:first-child {
  background: url("/assets/blq-orbit-blocks_grey.svg") no-repeat center center;
  width: 90px;
  padding: 0 15px;
}```

**Pull request merged**